bodhi/models/models.py

```diff
@@ -1044,6 +1044,7 @@ def set_request(self, db, action, username):
         topic = u'update.request.%s' % action
         if action is UpdateRequest.unpush:
             self.unpush()
+            self.request = None
```

Comment:
@ralphbean Yeah right. Although this already ensures when `action` is `UpdateRequest.unpush` the `request` is set to `None`, but it would make more sense to set request to `None` inside the definition of `unpush`. I'll try to do that :).